portfolio/src/main/webapp/index.html

```diff
@@ -118,8 +118,7 @@ <h3>Projects</h3>
 							<input type="submit" value="Submit">
               </form>
 					</div>
-					<div class="comments-item">
```

Comment:
Please reformat this file to use 2 spaces for indentations.

Reply:
I'm not sure why it looks like that here, on my end on CloudShell it is formatted correctly

Reply:
Was this updated?

Reply:
I didn't update it no, I really don't know how to make it look different on GitHub. Apparently some other interns were having the issue as well and can't make sense of it either. I have my tab size changed in CloudShell so I am using tabs not spaces

Reply:
Ack. This was resolved off line and via a new filed issue.